src/Stratis.Bitcoin.Features.Consensus/ConsensusFeature.cs

```diff
@@ -58,13 +59,15 @@ public class ConsensusFeature : FullNodeFeature
         }
 
         /// <inheritdoc />
-        public override void Initialize()
+        public override Task InitializeAsync()
```

Comment:
Why no logs here?

Reply:
no logger. 
decided to avoid touching too much- added logging only where the logger was already there

Reply:
This PR already touches 32 files so probably better will be to introduce logging to classes that lack it in another PRs